src/mfa/webauthn/types.ts

```diff
@@ -0,0 +1,11 @@
+import { User } from '../../user/types';
+
+export type WebAuthnDevice = {
+    id: number,
+    user: User,
+    credentialID: string,
```

Comment:
Could this be `credentialId`?

Reply:
I did this so the type format would match the library. Specifically, its when doing this:
https://github.com/curveball/a12n-server/pull/178/files#diff-7acaf47777dd5c150438c29e0cd66842R50

`verifyAssertionResponse` 's single param is an object 
https://github.com/MasterKale/SimpleWebAuthn/blob/master/packages/server/src/assertion/verifyAssertionResponse.ts#L35
and the `authenticator` field is this type:
https://github.com/MasterKale/SimpleWebAuthn/blob/master/packages/server/src/assertion/verifyAssertionResponse.ts#L16
`AuthenticatorDevice` has a `credentialID` field
https://github.com/MasterKale/SimpleWebAuthn/blob/master/packages/typescript-types/src/index.ts#L105

I can definitely change it. `credentialID` is *not* consistent with the rest of the application. I was just being lazy.